packages/plugins/plugin-table/src/table-model.ts

```diff
@@ -33,6 +34,13 @@ export type TableModelProps = {
   rowSelection?: number[];
 };
 
+// TODO(Zan): Is there a better place for this to live?
+export const columnSettingsButtonAttr = 'data-table-column-settings-button';
+const columnSettingsButtonClasses = 'ch-button is-6 pli-0.5 min-bs-0 absolute inset-block-1 inline-end-2';
+const columnSettingsIcon = 'ph--caret-down--regular';
+const columnSettingsButtonHtml = (columnId: string) =>
```

Comment:
we prob want a util for this in react-ui (e.g., at least factoring out the svg).